### DISCUSSION TITLE
I need help

### DISCUSSION THREAD

**Discussion by stuartcatt** [3kh0/ext-remover#1059] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
how do I stop go guaridian from sending me notifications
![Screenshot 2024-05-13 12 57 39 PM](https://github.com/3kh0/ext-remover/assets/168027816/fe056531-1761-4019-a0bd-0e6b5e46212c)

**Comment by AshtonDavies** [comment, marked as answer]
Click the time at the edge of the shelf > Notifications, then flip off the GoGuardian switch, and no, I mean click it, don't flip it off with your finger.

I do feel like you made that notification with Swamp although they patched that. Maybe your teacher is freaky.

**Reply by LizzyEll** [reply]
chrome://notification-tester/
